packages/client/src/client/helper.js

```diff
@@ -70,23 +70,37 @@ export class ClientHelper {
    */
   constructor({ platformUrl, appName, forceHttps = false, authentication, resource = null, transports = Transports }) {
     // Merge config with overloaded environment
-    const options = merge(
+    /**
+     * @access private
+     * @type {Object}
+     */
+    this.options = merge(
       {
         platformUrl,
         appName
       },
       transports.getOverloadedConfigFromEnvironment()
     );
     // Validate mandatory parameters
-    const mandatory = ['appName', 'platformUrl'].filter((property) => !options[property]);
+    const mandatory = ['appName', 'platformUrl'].filter((property) => !this.options[property]);
     if (mandatory.length) {
       throw new Error(`Missing mandatory parameter(s) ${mandatory.join(', ')}`);
     }
+    /**
+     * @access private
+     * @type {boolean}
+     */
+    this.forceHttps = forceHttps;
```

Comment:
Why add 2 new properties forceHttps and transports? 
They are already available via options properties

Reply:
Fixed